### DISCUSSION TITLE
Welcome to MLflow discussions!

### DISCUSSION THREAD

**Discussion by BenWilson2** [mlflow/mlflow#5565] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about. Seriously. Anything is fair game.
  * Share ideas. Share thoughts. Share things you wished this package would do but doesn't currently do.
  * Engage with other community members.
  * Welcome others, stay open-minded, and treat others with respect. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SommerChase** [comment]
Hello! My name is Chase Sommer and I am a Business Analyst who is learning ML. MLflow comes up a lot in a community I'm in ([MLOps.community](https://mlops-community.slack.com/)), so I'd love to learn more about it - even though I'm still a `noob `

**Reply by SommerChase** [reply]
Also, not to be creepy, but @BenWilson2 we live super close!!

**Reply by BenWilson2** [reply]
Want to be weirded out even more? Guess where I worked as a NitrideEpi MOCVD Engineer in a previous life?

**Reply by SommerChase** [reply]
Lol no way!! I'm interested in how you went from engineering semiconductors to ML software 🤔 Cool stuff

**Comment by BenWilson2** [comment]
@SommerChase I think a really good place to start is by running through the examples. If you want a full 'setup guide' about how to build a conda environment that has all of the packages you'll need without having to hunt for them, give a look at the [contributing guide](https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.rst). After you have the environment set up, feel free to run some of the [examples](https://github.com/mlflow/mlflow/tree/master/examples) so that you can see what the benefits are for experiment tracking. 
If you get stuck on anything, feel free to reach out on here and we'll help you along the way!

**Reply by SommerChase** [reply]
Will do! I'll test it out later this week/weekend.

**Comment by nihalgurjar47** [comment]
how to store plotly bar chart  in log_metric of mlflow?

**Reply by BenWilson2** [reply]
charts and any other image artifacts need to be stored with a log_artifact() call. `log_metric[s]()` is for numeric values only.

**Comment by nihalgurjar47** [comment]
Thanks for the confirmation.
________________________________
